### ISSUE TITLE
How to get command line help for flags, etc.

### ISSUE THREAD

**Issue by erichgoldman** [DDoSolitary/LxRunOffline#65]
Hello,

I am using version `LxRunOffline v3.3.1` installed by chocolatey. On the wiki I do not see detailed documentation and I have been googling to find examples for some basics. I was looking for more detailed help when running on the command line by running something like `LxRunOffline help run` but the command line reports back: "[ERROR] The action "help" is not recognized." I see in [another issue](https://github.com/DDoSolitary/LxRunOffline/issues/24#issuecomment-364889648) that this should in fact be the format.

I've also seems like some wiki pages have gone missing? I cloned the wiki repo and it looks like many pages were removed, is that for an older version only?

**Comment by DDoSolitary** [comment]
The comment in that issue applies to an older version. Simply run `LxRunOffline run` and you'll get what you want.

**Comment by DDoSolitary** [comment]
The wiki page about installing Alpine Linux was removed because the related bugs have been fixed in win10's 1809 release (I just forgot to update the reference). The one about creating shortcuts was removed because it's now directly supported by the program. There others are instructions on downloading necessary files for different distros and have been unified to the Home page.

**Comment by erichgoldman** [comment]
Thanks for the update, when I run `LxRunOffline run` I get the output "[ERROR] the option '-n' is required but missing". I was hoping for more verbose output and to see if there were optional flags - e.g., I don't see anything that states that `-n` means name.

**Comment by DDoSolitary** [comment]
That's strange, it should have output meaning of all possible flags. I'll look into this.

**Comment by DDoSolitary** [comment]
It turned out to be a regression introduced by 5f8419df9951d8291b2b613b4fb1a9bba3639215 and I just fixed. Please try the attached version and let know if it works, thanks. (The automated build on AppVeyor is taking longer than expected so I just uploaded the version I built locally.
[Release.zip](https://github.com/DDoSolitary/LxRunOffline/files/2608699/Release.zip)

**Comment by erichgoldman** [comment]
Hi, for the release here it seems to now show more details - given: `LxRunOffline.exe run`
```
[ERROR] the option '-n' is required but missing

Options:
  -n arg                Name of the distribution
  -c arg                The command to run. Launch default shell if not
                        specified.
  -w                    Don't use the working directory in Windows for the
                        Linux process.
```

However, I noticed in the attached that the "version" parameter is missing and not working: `[ERROR] The action "version" is not recognized.`

It looks like everything else is the same except the missing version command.

**Comment by DDoSolitary** [comment]
It is expected. `version` is only available in executables automatically built on AppVeyor, for which I wrote extra script to retrieve version info from git.

**Comment by DDoSolitary** [comment]
Now the automated build has completed and you'll find the version command working in this build: https://ci.appveyor.com/api/buildjobs/ownsme4qmr2syxpn/artifacts/LxRunOffline-v3.3.1-2-g4bd26d1.zip

I'll release this fix as v3.3.2 some time later.
